backend/x86_binary_emitter.ml

```diff
@@ -1531,12 +1541,14 @@ let imm8_of_float_condition = function
   | NLEf -> 0x06
   | ORDf -> 0x07
 
-let emit_cmpsd b ~condition ~dst ~src =
+let emit_cmp_float ~width b ~condition ~dst ~src =
   match (dst, src) with
   | (Regf reg, ((Regf _ | Mem _ | Mem64_RIP _) as rm)) ->
     (* CMPSD xmm1, xmm2/m64, imm8 *)
```

Comment:
(The comment should probably be tweaked.)

Reply:
Fixed